vortexasdk/api/vessel.py

```diff
@@ -41,31 +61,30 @@ class Vessel(Node, FromDictMixin):
     imo: Optional[int] = None
     gross_tonnage: Optional[int] = None
 
-    scrubber: Optional[Scrubber] = None
-    flag: Optional[Flag] = None
+    scrubber: Optional[List[Scrubber]] = None
+    flag: Optional[List[Flag]] = None
     ice_class: Optional[str] = None
     propulsion: Optional[str] = None
 
 
-@dataclass(frozen=True)
-class VesselEntity(IDName):
+class VesselEntity(IDName, BaseModel):
     """
     A VesselEntity represents a vessel record used in CargoMovements and VesselMovements.
 
     [Vessel Entities Further Documentation](https://docs.vortexa.com/reference/intro-vessel-entities)
     """
 
     id: ID
```

Comment:
Probably not needed due to inheriting from `IDName`

Reply:
Fixed.